src/main/java/org/jvnet/hudson/update_center/Plugin.java

```diff
@@ -69,43 +67,23 @@
      * Latest version of this plugin.
      */
     public final HPI previous;
-    /**
-     * Confluence page of this plugin in Wiki.
-     * Null if we couldn't find it.
-     */
-    public final RemotePage page;
-
-    /**
-     * Confluence labels for the plugin wiki page.
-     * Null if wiki page wasn't found.
-     */
-    private String[] labels;
-    private boolean labelsRead = false;
```

Comment:
`labels` and `labelsRead` are still left in the upstream, but actually they are no longer used.
That looks rather a bug of the upstream. (actually not a bug as that doesn't cause problems, but I don't know an appropriate word for this situation)
Please consider to contribute also to the upstream if you can.